src/screentovnc.cpp

```diff
@@ -389,9 +412,28 @@ bool ScreenToVnc::event(QEvent *e)
         LOG() << "push frame to vnc buffer";
         FrameEvent *fe = static_cast<FrameEvent *>(e);
         Buffer *buf = fe->buffer;
-        QImage img = fe->transform == LIPSTICK_RECORDER_TRANSFORM_Y_INVERTED ? buf->image.mirrored(false, true) : buf->image;
+        QImage bufImg = fe->transform == LIPSTICK_RECORDER_TRANSFORM_Y_INVERTED ? buf->image.mirrored(false, true) : buf->image;
```

Comment:
I think you should be able to avoid having two `QImage`s in stack by overwriting the first one with the transformed image. `QImage::transformed()` claims to return copy of the `QImage` and that can be then assigned over the original.